src/lib/array.js

```diff
@@ -4,3 +4,5 @@ export const intersection = arrays => {
   rest = rest.map(x => new Set(x))
   return first.filter(x => rest.every(a => a.has(x)))
 }
+
+export const flatten = arr => arr.reduce((acc, val) => acc.concat(val), [])
```

Comment:
A `flat()` method already exists as a native feature in ES6.

see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat